### DISCUSSION TITLE
How to set a backup node?

### DISCUSSION THREAD

**Discussion by austinguish** [argoproj/argo-workflows#5393] [Q&A] [answered]
Hi, everyone. When I set a DAG with the argo-workflows, I have a question that how to set a node as a backup when some nodes dead(get some errors).
For example:
          A 
   /       \      \
B       C     Back
  \       /        /
         D
only the C get error, the Back instead of.
Thanks!!!

**Comment by terrytangyuan** [comment, marked as answer]
Exit handlers might be what you need, see an example [here](https://github.com/argoproj/argo-workflows/tree/master/examples#exit-handlers). You can reference the status of nodes in a variable `{{workflow.status}}` as part of the `when` condition. There are also examples to do this at either step or DAG level:
* https://github.com/argoproj/argo-workflows/blob/master/examples/exit-handler-dag-level.yaml
* https://github.com/argoproj/argo-workflows/blob/master/examples/exit-handler-step-level.yaml

**Reply by austinguish** [reply]
Thank u, I'll try it!
